plutus-conformance/test-cases/uplc/evaluation/term/constant-case/bool/bool-04/bool-04.uplc.budget.expected

```diff
@@ -0,0 +1 @@
+evaluation failure
```

Comment:
That's weird. A failing program still consumes budget and we should report it. I do of course understand that it's not a problem with this PR.